frontend/src/pushapi.js

```diff
@@ -21,49 +17,16 @@ angular.module('clouway-push', [])
     };
 
     /**
-     * Set a method to call for opening of connection.
-     * The method must return a promise.
+     * Sets the backend url connection.
      *
-     * @param {function(subscriber)} method method that will be called for opening of connection.
+     * @param url the new url
      * @returns {*} the provider instance for chaining purposes.
      */
-    this.openConnectionMethod = function (method) {
-      this.connectionMethods.connect = method;
+    this.backendServiceUrl = function(url) {
```

Comment:
The reason I made it so that the whole methods to be configurable is that the frontend is not so strictly tied to our backend api.
Could you tell what your motivation behind this change is?